test/lint/test_runner/src/main.rs

```diff
@@ -512,6 +522,44 @@ sourced files to the exclude list.
     }
 }
 
+fn lint_trailing_newline() -> LintResult {
+    let files = check_output(
+        git()
+            .args([
+                "ls-files", "--", "*.py", "*.cpp", "*.h", "*.md", "*.rs", "*.sh", "*.cmake",
+            ])
+            .args(get_pathspecs_default_excludes()),
+    )?;
+    let mut missing_newline = false;
+    for path in files.lines() {
```

Comment:
would it help if we used `.par_iter()` here instead (not sure if that would require adding a new lib or not, but seems like a naturally parallelizable task)?

Reply:
I haven't tested it, if you say it's fast enough, I believe you